linkcheck/configuration/confparse.py

```diff
@@ -201,20 +231,36 @@ def check_password_readable(self, section, fields):
             return
         fn = self.read_ok[0]
         if fileutil.is_accessable_by_others(fn):
-            log.warn(LOG_CHECK, "The configuration file %s contains password information (in section [%s] and options %s) and the file is readable by others. Please make the file only readable by you.", fn, section, fields)
+            log.warn(
+                LOG_CHECK,
+                "The configuration file %s contains password information (in "
+                "section [%s] and options %s) and the file is readable by "
+                "others. Please make the file only readable by you.",
```

Comment:
Out of scope for this PR, but huh!  this message is not translated!  It should be.